siriuspy/siriuspy/devices/injsys.py

```diff
@@ -591,16 +472,33 @@ def __init__(self, hltiming=None):
             self._on_values[dev] = {
```

Comment:
the loop here should be on `self._llrf_devs`, not `devices`

Reply:
right, done!